coopr-standalone/bin/coopr.sh

```diff
@@ -345,7 +342,7 @@ greeting () {
 
 stop () { provisioner stop; server stop; ui stop; }
 
-start () { server start && ui start && provisioner start && load_defaults && greeting; }
+start () { server start && ui start && provisioner register && provisioner start && load_defaults && greeting; }
```

Comment:
does plugin registration depend on server start or is this just a cleaner way to start all of these?